### DISCUSSION TITLE
功能建议：仅对部分生词进行翻译

### DISCUSSION THREAD

**Discussion by simonwan1980** [immersive-translate/immersive-translate#378] [Ideas]
场景：出于学习英语的目的，并不希望对整个网页进行全文翻译，只是希望能对某些生词进行翻译即可。
参考“听阅”APP，用户可以选择仅对部分生词（比如按照四六级区分）进行翻译。而且对于已经熟悉的单词也可以手动将改词设置为“已会”不再进行翻译

**Comment by theowenyoung** [comment]
这个功能以后可能会支持～

暂时推荐这个插件： https://relingo.net/en/index 

它可以完美实现你的需求～

**Comment by simonwan1980** [comment]
Thanks

**Comment by simonwan1980** [comment]
我用了几天，你推荐的这个插件还不算“完美”，有如下几点问题：
1. *****对于多义词，该插件只备注一种词义，虽然通过“展开”的形式来能看多个词义，但是并不能手工将“备注”的词义切换想要的词义。比如train，“火车”我已知道，但是“训练”还不知道，因此希望能够让这个词继续保持“生词”状态，但是备注词义能选择为“训练”。听阅，有这个功能；
2. ****不能手工修改或者增加词义；可以认为这是对问题1的一种延伸；
3. ****不识别词组，只识别单词。比如turn on
4. *个人感觉还是听阅那种在单词下面“夹行”加注释的方式，读起来更流畅，气更顺。

> 这个功能以后可能会支持～
> 
> 暂时推荐这个插件： https://relingo.net/en/index
> 
> 它可以完美实现你的需求～
